### ISSUE TITLE
Using more general type ts.Node instead of ts.SourceFile

### ISSUE THREAD

**Issue by manfredsteyer** [phenomnomnominal/tsquery#5]
Can we use ``ts.Node`` instead of ``ts.SourceFile`` for the first parameter in ``tsquery.query`` or ``tsquery``?

This would allow us to create a query for a specific node that is the result of another query.

A first test where I just casted to <any> seems to be successful:

```
node = /* ... result of another tsquery ... */
result = tsquery(<any>node.parent, 'ImportSpecifier Identifier[text="Component"]');
```

**Comment by phenomnomnominal** [comment]
Oh yeah that’s a great call, that’s how esquery works. I’ll do that right now.

**Comment by phenomnomnominal** [comment]
Fixed with https://github.com/phenomnomnominal/tsquery/commit/8debd40549ef92e08c7136c64b6ad0344ff7bdd8 and published @1.0.1

**Comment by manfredsteyer** [comment]
Thx!
